fbw-a32nx/src/systems/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -171,9 +172,9 @@ const ApuBleed = ({ x, y } : ComponentPositionProps) => {
                 <text
                     x={44}
                     y={48}
-                    className={`FontLarger Right ${adir1ModeSelectorKnob === 1 ? 'Green' : 'Amber'}`}
+                    className={`FontLarger Right $className= ${apuBleedPressAvailable ? 'Green' : 'Amber'}`}
```

Comment:
This looks wrong

Reply:
This was @ tracernz recommended fix from https://github.com/flybywiresim/aircraft/pull/8051  
vebatim suggestion is below

`className={`FontLarger Right ${apuN.isNormalOperation() && adir1ModeSelectorKnob === 1 ? 'Green' : 'Amber'}`}
                    className={`FontLarger Right ${apuBleedPressAvailable ? 'Green' : 'Amber'}`}`

Also regarding these fixes I thought I had made a new branch for #8051 for these fixes.. not sure how this happened

Reply:
Please go over the code. You pasted it wrong. The syntax is invalid.